src/OrchardCore/OrchardCore.Abstractions/Extensions/DocumentJsonSerializerOptionsConfiguration.cs

```diff
@@ -16,7 +16,7 @@ public DocumentJsonSerializerOptionsConfiguration(IOptions<JsonDerivedTypesOptio
 
     public void Configure(DocumentJsonSerializerOptions options)
     {
-        // Do not use the 'Merge' extension to avoid populating unwanted properties (e.g., Encoder, NumberHandling, AllowTrailingCommas).
+        // Do not use the 'Merge' extension to avoid populating unwanted properties (e.g., Encoder, NumberHandling, TypeInfoResolver).
```

Comment:
```suggestion
        // Do not use the 'Merge' extension to avoid populating unwanted properties (e.g., Encoder, NumberHandling, AllowTrailingCommas, TypeInfoResolver).
```

Reply:
Hihi, no. You're setting that property 5 lines below ;-)

Reply:
I see. I am okay with your PR and feel free to merge it. However, I think we should actually change this settings in the document settings. These settings should not be copied from Base and should be strictly set here. They could also slightly improve performance.

```
options.SerializerOptions.AllowTrailingCommas = false;
options.SerializerOptions.WriteIndented = false;
options.SerializerOptions.NumberHandling = JsonNumberHandling.Strict;
```

Reply:
Yeah, maybe. For now let's not mess with this again ;)